### PR TITLE
DB-4 add SQL script for people table

### PR DESCRIPTION
**Pull request by NikShein**

### CODE REVIEW

File: SQL/people.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE People 
+(
+    Id SERIAL PRIMARY KEY,
+    FirstName CHARACTER VARYING(30) NOT NULL,
+    LastName CHARACTER VARYING(30) NOT NULL,
+    DateOfBirth INTEGER
```
Comment (nikitabls) [inline SQL/people.sql:6]: It is better to use the Date type for storing dates

File: SQL/people.sql [hunk 2]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE People
```
Comment (nikitabls) [inline SQL/people.sql:1]: It is also better to name SQL scripts more verbosely, so we can know what this script is doing:
For example, this one is better to be named "create_people_table.sql"

File: SQL/people.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+CREATE TABLE People 
+(
+    Id SERIAL PRIMARY KEY,
```
Comment (belousnikita) [inline SQL/people.sql:3]: Why ID is a SERIAL type?

File: SQL/people.sql (hunk 1 above)
Comment (belousnikita) [inline SQL/people.sql:6]: For date it is better to use Date types

**Comment by belousnikita** [comment]
Please use more verbose names for files

**Review by belousnikita — CHANGES_REQUESTED** [review]
Please see comments

File: SQL/people.sql (hunk 3 above)
Comment (NikShein) [reply SQL/people.sql:3]: `PostgreSQL has a special kind of database object generator called SERIAL. It is used to generate a sequence of integers which are often used as the Primary key of a table. When creating a table, this sequence of integers can be created as follows: CREATE TABLE table_name( id SERIAL );`

[PostgresSQL - Serial Type](https://www.geeksforgeeks.org/postgresql-serial/#:~:text=PostgreSQL%20has%20a%20special%20kind,Primary%20key%20of%20a%20table.&text=When%20creating%20a%20table%2C%20this,TABLE%20table_name(%20id%20SERIAL%20)%3B)

**Review by belousnikita — APPROVED** [review]

**Pull request merged**